### ISSUE TITLE
Changes

### ISSUE THREAD

**Issue by willghatch** [willghatch/racket-rash#9]
FYI, to anybody watching this repo, I've got some fairly major changes coming.  I did say it was unstable.

* `|>` will no longer turn ports to strings, instead `|>>` will do that.
* String reading at expansion time is cool, but it is not really necessary, and is mostly a holdout from earlier designs with poor separation between macros and reading.  Also, I've given in to having more normal ascii delimiters available.  So now `{}` will switch the reader to line-mode, and it will happen as a part of the original read phase, not mixed with expansion.  However, the `rash` macro will still allow strings.  So you can do it either way.  `{}` will not wrap the line-mode code with any configuration, so eg. if you are at the top level and the default output for processes is `current-output-port`, it will still be inside `{}`.  `#{}` wraps it with the default `rash` macro settings, IE an empty input port, output converted to a string, etc.

So basically, you might have to change some pipelines that use `|>`.  Basically this makes things more consistent when you want to eg. read process output as something other than a string.

Those are the big changes that I think people following the repo might want to know about.  I haven't merged the changes into master yet, but I will in the next couple days.  Probably when I write some documentation to go with the changes.
